src/vm/gcenv.os.cpp

```diff
@@ -205,6 +246,33 @@ bool GCToOSInterface::VirtualRelease(void* address, size_t size)
     return !!::ClrVirtualFree(address, 0, MEM_RELEASE);
 }
 
+// Commit virtual memory range.
+// Parameters:
+//  size      - size of the virtual memory range
+// Return:
+//  Starting virtual address of the committed range
+void* GCToOSInterface::VirtualReserveAndCommitLargePages(size_t size)
+{
+    LIMITED_METHOD_CONTRACT;
+
+#if !defined(FEATURE_PAL)
+    if (!g_SeLockMemoryPrivilegeAcquired)
+    {
+        if (!InitLargePagesPrivilege())
+        {
+            return nullptr;
+        }
+
+        g_SeLockMemoryPrivilegeAcquired = true;
+    }
+
+    auto largePageMinimum = GetLargePageMinimum();
```

Comment:
See my previous comment on using `auto`

Reply:
right, again, I don't see any need for an auto var here.

---
In reply to: [271181210](https://github.com/dotnet/coreclr/pull/23251#discussion_r271181210) [](ancestors = 271181210)